ruby/private/bundle/create_bundle_build_file.rb

```diff
@@ -247,6 +251,9 @@ def remove_bundler_version!
   end
 
   def register_gem(spec, template_out, bundle_lib_paths, bundle_binaries)
+    # Do not register local gems
+    return if spec.source.path?
```

Comment:
Can you explain that line?

Reply:
It's not possible to register local gems as they are not linking to `external/bundle`. This guard just prevents the code from even trying to do so.